### DISCUSSION TITLE
For consideration: Link to OpenWrt installation guide

### DISCUSSION THREAD

**Discussion by jamesmacwhite** [AdguardTeam/AdGuardHome#4056] [General]
Hi,

The OpenWrt community has been working hard to create an official wiki page for AdGuard Home here:

https://openwrt.org/docs/guide-user/services/dns/adguard-home

There has been a lot of excellent community guides written on the OpenWrt forum, but there's been a recent effort to take all the best bits from these contributions for the official wiki page, incorporating all the key information and best configuration practices.

OpenWrt has an official package that can be installed via opkg for version 21.02 onwards to simplifying installation. This also bundles an init script to take care of running AdGuard Home as a service, which the wiki page promotes as the easy installation method.

Would the AdGuard team consider linking to the OpenWrt wiki page above as part of the "Installing AdGuard Home" section?

Thanks!

**Comment by ainar-g** [comment]
Thanks to the OpenWrt community and apologies for the late response!  We have now added the link to the [home page] of our Wiki.

[home page]: https://github.com/AdguardTeam/AdGuardHome/wiki

**Comment by jamesmacwhite** [comment]
@ainar-g And sorry for my late response! Thank you for adding a link to your main wiki! The OpenWrt community will do it's best to keep the wiki maintained for use on OpenWrt routers.
